### DISCUSSION TITLE
Can Taskbar Thumbnail Reorder save the order I have set?

### DISCUSSION THREAD

**Discussion by robmooar** [ramensoftware/windhawk-mods#555] [Q&A] [answered]
Hi all

Firstly, thank you for the fantastic piece of software.  This has made my life so much easier.

I was wondering if there is a way to save the order of the tabs in the Taskbar Thumbnail Reorder mod so that after restarting the computer/application that the reordering was done on, it would remain in the order I set?  I thought it used to save the order in 7+ Taskbar Tweaker, but am wondering if this is a limitation of Windhawk (which is ok, I can live with that).

Thanks again, and thanks in advance for any advice you can give.

Cheers

Rob

**Comment by m417z** [comment, marked as answer]
Hi Rob,
It isn't supported by 7+ Taskbar Tweaker, too. It was suggested a couple of times, see my replies for an explanation for why it's tricky to implement:
* https://ramensoftware.com/contact/comment-page-3#comment-7707
* https://ramensoftware.com/contact/comment-page-5#comment-10884

**Reply by robmooar** [reply]
Thanks for the update.  That makes perfect sense, esp. the second link.  It is what it is.. reordering after a reboot is a small price to pay for the convenience it has between times.

Thanks again, and keep up the great work.

Rob
